### ISSUE TITLE
[Bug]: logo64.png no such file or directory.

### ISSUE THREAD

**Issue by ARelaxedScholar** [tweag/jupyenv#537]
### Current Behavior

running nix run with a project setup to use poetry leads to :

`warning: ignoring untrusted substituter 'https://tweag-jupyter.cachix.org', you are not a trusted user.
Run `man nix.conf` for more information on the `substituters` configuration option.
error (ignored): error: end of string reached
error:
       … while calling the 'getAttr' builtin

         at /builtin/derivation.nix:19:19: (source not available)

       … while calling the 'derivationStrict' builtin

         at /builtin/derivation.nix:9:12: (source not available)

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: getting status of '/nix/store/mdi2dk2hmvm4hv8sf01x88riqc4av2ay-source/affective_dependencies/logo64.png': No such file or directory
`

### Expected Behavior

It should run and allow me to run JupyterLab with my custom kernel.

### Steps To Reproduce

Basically follow he poetry part of this: https://jupyenv.io/documentation/how-to/
1. run `nix flake init --template github:tweag/jupyterWith`
2. create a directory in my case `affective_dependencies`
3. create your pyproject.toml and run `poetry lock` in there
4. make sure kernels.nix contains a reference to your poetry directory with `projectDir`
5. run `nix run` in the top directory

### OS

```shell
- system: `"x86_64-linux"`
 - host os: `Linux 6.1.78, NixOS, 23.11 (Tapir), 23.11.20240326.4473351`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

### Version

main

### Additional Context

I would use the built in core but I need ts2vg which is not part of nixpkgs.

### Relevant log output

```shell
warning: ignoring untrusted substituter 'https://tweag-jupyter.cachix.org', you are not a trusted user.
Run `man nix.conf` for more information on the `substituters` configuration option.
warning: creating lock file '/home/arelaxedscholar/ドキュメント/Programming/Jupyter Notebooks/playing_with_ppg/flake.lock'
error:
       … while calling the 'getAttr' builtin

         at /builtin/derivation.nix:19:19: (source not available)

       … while calling the 'derivationStrict' builtin

         at /builtin/derivation.nix:9:12: (source not available)

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: getting status of '/nix/store/b1jl6n5a2yh1pl6i0xqj09hmp24mddhn-source/affective_dependencies': No such file or directory
```

**Comment by kyp0717** [comment]
What is the status of this bug?  I am encountering the same issue.  I would like to try to fix it but need some help to get started.

**Comment by GTrunSec** [comment]
closes by #524
